openstack/compute/v2/servers/fixtures.go

```diff
@@ -1,7 +1,7 @@
-// +build fixtures
```

Comment:
`go test` has a bug where unless the `package` statement is the top, it won't import variables into other files in the same package.